config/src/test/java/org/springframework/security/config/web/server/OneTimeTokenLoginSpecTests.java

```diff
@@ -280,6 +286,36 @@ Please provide it as a bean or pass it to the oneTimeTokenLogin() DSL.
 					""");
 	}
 
+	@Test
+	void oneTimeTokenWhenCustomTokenExpirationTimeSetThenAuthenticate() throws Exception {
```

Comment:
Will you please add a test that confirms in isolation that the filter uses a custom a request resolver?